frontend/packages/console-plugin-sdk/src/typings/dashboards.ts

```diff
@@ -37,12 +34,24 @@ namespace ExtensionProperties {
      * Response is then parsed by `healthHandler`.
      */
     fetch?: (url: string) => Promise<R>;
+
+    /** Resolve the subsystem's health */
+    healthHandler: (response: R) => SubsystemHealth;
   }
 
   export interface DashboardsOverviewHealthPrometheusSubsystem
-    extends DashboardsOverviewHealthSubsystem<any> {
+    extends DashboardsOverviewHealthSubsystem {
     /** The Prometheus query */
     query: string;
+
+    /** Resource which will be fetched and passed to healthHandler  */
+    resource?: FirehoseResource;
+
+    /** Resolve the subsystem's health */
+    healthHandler: (
+      response,
```

Comment:
Are we OK with `response` having `any` type here?

(`DashboardsOverviewHealthURLSubsystem` allows response type parametrization via generics.)

Reply:
it is actually a `PrometheusResponse` type, added